snapcraft/tests/unit/test_lxd.py

```diff
@@ -537,6 +537,52 @@ def make_containerbuild(self):
                            project_options=self.project_options,
                            remote=self.remote)
 
+    @patch('snapcraft.internal.lxd.Containerbuild._container_run')
+    @patch('snapcraft.internal.common.is_snap')
+    def test_inject_snap_existing_container(self,
+                                            mock_is_snap,
+                                            mock_container_run):
+        mock_is_snap.side_effect = lambda: True
+        mock_container_run.side_effect = lambda cmd, **kwargs: cmd
```

Comment:
The real function doesn't actually return anything... is this really what you want?